include/inexor/vulkan-renderer/wrapper/command_pool.hpp

```diff
@@ -26,10 +26,14 @@ class CommandPool {
     CommandPool(const VkDevice device, const std::uint32_t queue_family_index);
 
     ~CommandPool();
-
+    
```

Comment:
Weird white space, clang-format should fix this though.

Reply:
Done.